rest_framework/management/commands/generateschema.py

```diff
@@ -1,41 +1,32 @@
 from django.core.management.base import BaseCommand
 
-from rest_framework.compat import coreapi
-from rest_framework.renderers import (
-    CoreJSONRenderer, JSONOpenAPIRenderer, OpenAPIRenderer
-)
-from rest_framework.schemas.generators import SchemaGenerator
+from rest_framework.compat import yaml
+from rest_framework.schemas.generators import OpenAPISchemaGenerator
+from rest_framework.utils import json
 
 
 class Command(BaseCommand):
     help = "Generates configured API schema for project."
 
     def add_arguments(self, parser):
-        parser.add_argument('--title', dest="title", default=None, type=str)
+        parser.add_argument('--title', dest="title", default='', type=str)
         parser.add_argument('--url', dest="url", default=None, type=str)
         parser.add_argument('--description', dest="description", default=None, type=str)
-        parser.add_argument('--format', dest="format", choices=['openapi', 'openapi-json', 'corejson'], default='openapi', type=str)
+        parser.add_argument('--format', dest="format", choices=['openapi', 'openapi-json'], default='openapi', type=str)
```

Comment:
We'll prob. want a urlconf parameter too. Should be able to draft something if you're interested in

Reply:
Hey @xordoquy. Yes! 😀 Please! — all input REALLY welcome at this point: it's shaping up nicely but a lot to do still... _Many hands make light work_ we say.